src/styles/typography/index.njk

```diff
@@ -79,12 +79,19 @@ The base font size for paragraphs is 18px.
 }}
 
 ## Text indent
-Use when providing an address or contact information
+Use when providing contact information.
 
 {{
     patternlibExample({"path": "styles/typography/text-indent/examples/text-indent/index.njk", "componentFolderPath": "components/text-indent"})
 }}
 
+## Addresses
```

Comment:
If we do want to differentiate this from the Address pattern we could change this line to "Address output" which would also update the nav link

Reply:
I think we just need to change the folder name of the current component from `addresses` to `address-input`.  The frontmatter can stay as it is.

Reply:
Fair enough, may as well also do that in this PR then